Resources/Prototypes/Entities/Structures/Machines/bombs.yml

```
@@ -0,0 +1,148 @@
+- type: entity
+  abstract: true
+  parent: BaseStructure
+  id: BaseHardBomb
+  description: Just keep talking and nobody will explode.
+  name: hardbomb
+  components:
+    - type: Appearance
+    - type: WiresVisuals
+    - type: InteractionOutline
+    - type: UserInterface
+      interfaces:
+        - key: enum.WiresUiKey.Key
+          type: WiresBoundUserInterface
+    - type: Wires
+      LayoutId: Defusable
+      alwaysRandomize: true
+    - type: Defusable
+    - type: Explosive
+      explosionType: Default
+      totalIntensity: 20.0
+      intensitySlope: 5
+      maxIntensity: 4
+    - type: ExplodeOnTrigger
+    - type: OnUseTimerTrigger
+      delay: 90
+      delayOptions: [90, 120, 150, 180, 210, 240, 270, 300]
+      initialBeepDelay: 0
+      beepSound: /Audio/Machines/Nuke/general_beep.ogg
+    - type: Anchorable
+      delay: 5
+    - type: Physics
+      bodyType: Static
+    - type: Transform
+      noRot: true
+    - type: Fixtures
+      fixtures:
+        fix1:
+          shape:
+            !type:PhysShapeAabb
+            bounds: "-0.45,-0.45,0.45,0.45"
+          density: 190
+          mask:
+            - MachineMask
+          layer:
+            - MachineLayer
+    - type: WiresPanel
+    - type: Damageable
+      damageContainer: Inorganic
+      damageModifierSet: Metallic
+    - type: Destructible
+      thresholds:
+        - trigger:
+            !type:DamageTrigger
+            damage: 100
+          behaviors:
+            - !type:DoActsBehavior
+              acts: ["Destruction"]
+            - !type:ExplodeBehavior
+            - !type:PlaySoundBehavior
+              sound:
+                path: /Audio/Effects/metalbreak.ogg
+    - type: GuideHelp
+      openOnActivation: true
+      guides:
+        - Defusal
+
+- type: entity
+  parent: BaseHardBomb
+  id: TrainingBomb
+  name: training bomb
+  description: A bomb for dummies, manual not included.
+  components:
+    - type: Wires
+      LayoutId: DefusableTraining
```

```suggestion
      LayoutId: TrainingDefusable
```

cmon now....... no testin........